bundle/views/home.yaml

```diff
@@ -107,29 +108,43 @@ definition:
                 - uesio/io.text:
                     text: The User Experience Studio (ues.io) is a comprehensive, open source, no-code/pro-code, AI enabled platform. It offers the freedom to extend your application functionality with your own code. Only start paying for usage when your users are engaging.
                     uesio.variant: uesio/crmsite.herosubtext
-    - uesio/crmsite.splithero:
-        leftBackground: slate-100
-        rightBackground: slate-50
-        left:
-          - uesio/io.box:
-              uesio.variant: uesio/crmsite.textsection
-              components:
-                - uesio/io.text:
-                    uesio.variant: uesio/crmsite.subheader
-                    text: Prospecting
-                - uesio/io.text:
-                    uesio.variant: uesio/crmsite.halfhero
-                    text: Qualify and deduplicate leads.
-                - uesio/io.text:
-                    text: Take your sales game to the next level with optional AI integrations. Streamline your sales process, gain valuable insights, and boost overall efficiency.
-                    uesio.variant: uesio/crmsite.herosubtext
-        right:
-          - uesio/io.image:
-              file: uesio/crmsite.mockup_lead_convert
-              uesio.variant: uesio/crmsite.splithero
+                - uesio/crmsite.splithero:
+                    leftBackground: slate-100
+                    rightBackground: slate-50
+                    left:
+                      - uesio/io.box:
+                          uesio.variant: uesio/crmsite.textsection
+                          components:
+                            - uesio/io.text:
+                                uesio.variant: uesio/crmsite.subheader
+                                text: Prospecting
+                            - uesio/io.text:
+                                uesio.variant: uesio/crmsite.halfhero
+                                text: Qualify and deduplicate leads.
+                            - uesio/io.text:
+                                text: Take your sales game to the next level with optional AI integrations. Streamline your sales process, gain valuable insights, and boost overall efficiency.
+                                uesio.variant: uesio/crmsite.herosubtext
+                    right:
+                      - uesio/io.image:
+                          file: uesio/crmsite.mockup_lead_convert
+                          uesio.variant: uesio/crmsite.splithero
+                          uesio.styleTokens:
+                            root:
+                              - md:translate-x-1/3
+        header:
+          - uesio/core.view:
+              view: uesio/crmsite.header
               uesio.styleTokens:
                 root:
-                  - md:translate-x-1/3
+                  - sticky
```

Comment:
are both of these styling nodes needed to make the header sticky? I would think you'd only need one.

Reply:
Not sure. I just added the sticky style.